src/System.Drawing.Design.Primitives/src/System/Drawing/Design/ToolboxComponentsCreatedEventArgs.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Drawing.Design {
+    using System;
+    using System.ComponentModel;
+
+    /// <summary>
+    /// Provides data for the 'ToolboxComponentsCreatedEventArgs' event that occurs
+    /// when components are added to the toolbox.
+    /// </summary>
+    [System.Security.Permissions.PermissionSetAttribute(System.Security.Permissions.SecurityAction.InheritanceDemand, Name="FullTrust")]
+    [System.Security.Permissions.PermissionSetAttribute(System.Security.Permissions.SecurityAction.LinkDemand, Name="FullTrust")]
+    public class ToolboxComponentsCreatedEventArgs : EventArgs {
+        private readonly IComponent[] comps;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref='System.Drawing.Design.ToolboxComponentsCreatedEventArgs'
+        /// </summary>
+        /// <param name="components">components</param>
+        public ToolboxComponentsCreatedEventArgs(IComponent[] components) {
+            this.comps = components;
+        }
+
+        /// <summary>
+        /// An array storing the toolbox components.
+        /// </summary>
+        public IComponent[] Components {
+            get {
```

Comment:
Style guidelines - Brace on its own line

Reply:
You should be able to run [codeformatter](https://github.com/dotnet/codeformatter) to fix most of these.